gui/app/directives/controls/editable-tags.js

```diff
@@ -0,0 +1,101 @@
+"use strict";
+
+
+(function() {
+
+    const deepmerge = require("deepmerge");
+
+    angular
+        .module('firebotApp')
+        .component("editableTags", {
+            bindings: {
+                model: "=",
+                settings: "<",
+                onUpdate: '&'
+            },
+            template: `
+                <div>
+                    <div class="role-bar clickable" ng-repeat="item in $ctrl.model track by $index" ng-click="$ctrl.editItem($index)" aria-label="{{item + ' (Click to edit)'}}">
```

Comment:
This is a bit tricky for screen readers. With `ng-click` you define this as a button, but then there's also a button inside of that button that you can now not access. I'd suggest to put move this click from the container div to the span that contains the tag name, so that both buttons are accessible.

Reply:
Fixed